app/scheme.go

```diff
@@ -11,51 +11,56 @@ import (
 )
 
 func (a *App) GetScheme(id string) (*model.Scheme, *model.AppError) {
-	if err := a.IsPhase2MigrationCompleted(); err != nil {
```

Comment:
When you only return an error (or only use the error of the return), is really common to uses the single-line approach.